pkgs/development/libraries/libiio/default.nix

```diff
@@ -17,7 +18,17 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "lib" "dev" "python" ];
 
   nativeBuildInputs = [ cmake flex bison ];
-  buildInputs = [ libxml2 ];
+  buildInputs = [ libxml2 libusb1 ];
+
+  postPatch = ''
+    substituteInPlace libiio.rules.cmakein \
+      --replace /bin/sh ${runtimeShell}
```

Comment:
It is when you add libiio to NixOS' `services.udev.packages`. Otherwise you get the error:
```
Consider fixing the following udev rules:
/nix/store/xi7ik7ghd0sl5963d3dzqwmhkbnm7bhg-udev-rules/90-libiio.rules (originally from /nix/store/95a2gcs6ppf14jc5h4cymch13djrvmsd-libiio-0.18/etc/udev/rules.d/90-libiio.rules) contains references to /bin/sh.
```
which is actually an error ; and /bin/sh is impure anyway.